packages/apps/explorer/src/pages/index.tsx

```diff
@@ -1,10 +1,46 @@
+import type { NetworkInfo } from '@/__generated__/sdk';
+import { useNetworkInfoQuery } from '@/__generated__/sdk';
+import StatisticsStack from '@/components/statistics-stack/statistics-stack';
+import { LogoKdacolorLight } from '@kadena/react-icons/brand';
+import { Stack } from '@kadena/react-ui';
 import React from 'react';
 
+const formatStatisticsData = (
+  networkInfo: NetworkInfo | null | undefined,
+): { label: string; value: string }[] => {
+  if (!networkInfo) {
+    return [
+      { label: 'Est. Network Hash', value: '0 PH/s' },
+      { label: 'Total Difficulty', value: '0 EH' },
```

Comment:
Text such as "PH" and "EH" has to be calculated based on the length of the number. Check the explorer and compare the testnet and mainnet stats